tests/end2end/playwright/print.spec.js

```diff
@@ -43,103 +55,117 @@ test.describe('Print', () => {
     });
 
     test('Print requests', async ({ page }) => {
```

Comment:
I might get wrong, but this test can be split.

It makes the testing easier if each scenario are in their respectiv `test()`. If one fail, the others will run.